recipe/meta.yaml

```diff
@@ -12,18 +12,19 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
+  noarch: python
```

Comment:
Just as FYI, re-rendering is needed when switching to or from `noarch`. Though now we can ask a bot to do the re-rendering for us. 😄